cmd/manager/main.go

```diff
@@ -426,6 +426,7 @@ func setupWebhook(mgr manager.Manager, certRotation certificates.RotationParams,
 	})
 
 	if err != nil {
+		log.Error(err, "Timeout lapsed waiting for webhook certificate to be available", "path", keyPath)
```

Comment:
"elapsed"?
Let's also log the timeout value.

Reply:
english is hard